src/main/java/nextstep/users/domain/NsStudent.java

```diff
@@ -23,6 +35,17 @@ public Long getUserId() {
         return userId;
     }
 
+    public String getApplicationState() {
+        return applicationState.name();
+    }
+
+    public void changeStatus(ApplicationState applicationState) {
+        if (!ApplicationState.isPending(this.applicationState)) {
```

Comment:
```suggestion
        if (!this.applicationState.isPending()) {
```

이렇게 구현하는게 더 자연스러울것 같아요!